Snowcast/ResortsTableViewController.swift

```diff
@@ -1,21 +1,39 @@
+import PromiseKit
 import UIKit
+import CoreLocation
 
 class ResortsTableViewController: UITableViewController {
     
     let resortTableCell = "resortTableCell"
-    let showResortDetailSegue = "showResortDetailSegue"
-    
-    var resortService: ResortService?
-    var resorts: Array<ResortPreview> = []
+    var requestSnowData: RequestSnowData = RequestSnowData()
+    var resortService: ResortService = ResortService()
+    var locationService: UserLocation = UserLocation()
+    var selectedIndexPath : NSIndexPath?
```

Comment:
Colon Violation: When specifying a type, always associate the colon with the identifier